AWS/cloudformation-status.30s.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env /Users/sergio.pena/.asdf/installs/python/3.13.2/bin/python
```

Comment:
The below ensures python is picked up from either one's path, homebrew, or system installed.

```suggestion
#!/usr/bin/env -S PATH="${PATH}:/opt/homebrew/bin:/usr/local/bin" python3
```

Reply:
@sprak3000 thank you!

I totally forgot I hardcoded that